templates/patterns/page_header/pattern-page-header.html.twig

```diff
@@ -44,7 +44,7 @@
 {% if identity is not empty %}
   <div class="ecl-u-bg-blue ecl-u-pv-2xs">
     <div class="ecl-container ">
-      <h2 class="ecl-u-color-white ecl-u-type-heading-2">{{ identity }}</h2>
+      <h2 class="ecl-u-type-color-white ecl-u-type-heading-2">{{ identity }}</h2>
```

Comment:
Out of scope but ok to include it.

Reply:
After upgrading packages, the class `ecl-u-color-white` doesn't exist and replaced by `ecl-u-type-color-white` .
If we had visual tests, this upgrade will break the test. so one part is on the scope of the ticket.
;)